custom_components/ramses_cc/broker.py

```diff
@@ -92,15 +92,23 @@ async def start(self) -> None:
             client_state.get(SZ_SCHEMA, {}) if restore_schema else {},
         )
 
-        cached_packets = (
-            self._filter_cached_packets(
-                client_state.get(SZ_PACKETS, {}), restore_schema
-            )
-            if restore_state
-            else {}
-        )
+        @callback
```

Comment:
Do we need this decorator?

I never 100% understood what it's for, but as its only ever going to be synchronously invoked by our `start()` (and nor, say, `ceate_task()` / `call_soon()`), I don't believe it's needed.

Reply:
No, fair point. Hass will never be calling this directly so there should be no need.